sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -112,42 +136,89 @@ private static IReadOnlyList<CustomFormSubmodel> ConvertFromUnlabeled(KeysResult
                 subModels.Add(new CustomFormSubmodel(
                     $"form-{cluster.Key}",
                     default,
-                    fieldMap));
+                    fieldMap,
+                    model.ModelInfo.ModelId));
             }
             return subModels;
         }
 
         private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeled(Model model)
+        {
+            return new List<CustomFormSubmodel> {
+                new CustomFormSubmodel(
+                    $"form-{model.ModelInfo.ModelId}",
+                    model.TrainResult.AverageModelAccuracy,
+                    CalculateFieldMap(model.TrainResult),
+                    model.ModelInfo.ModelId) };
+        }
+
+        private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeledComposedModel(Model model)
+        {
+            var submodels = new List<CustomFormSubmodel>();
+
+            foreach (var trainResult in model.ComposedTrainResults)
+            {
+                submodels.Add(new CustomFormSubmodel(
+                    $"form-{trainResult.ModelId}",
```

Comment:
Same comment here. I don't remember what the naming convention is for composed model, though.

Reply:
I updated it to 1custom `prefix-{trainResult.modelId}` as that is all the information we have for the submodel for composed models.